docs/api.md

```diff
@@ -669,6 +677,7 @@ stream.chain(f):   -1--2-13---2-1-233|
 
 Note the difference between [`concatMap`](#concatmap) and [`chain`](#chain): `concatMap` concatenates, while `chain` merges.
 
+<!-- skip-example -->
```

Comment:
Yeah, not really sure the best way to handle it from markdown-test's perspective. In any case, I should probably reduce the times :)